src/essreflectometry/orso.py

```diff
@@ -142,6 +157,43 @@ def build_orso_data_source(
     )
 
 
+_CORRECTIONS_BY_GRAPH_KEY = {
+    ChopperCorrectedTofEvents[Sample]: 'chopper ToF correction',
+    FootprintCorrectedData[Sample]: 'footprint correction',
+    IofQ[Sample]: 'total counts',
+    SupermirrorCalibrationFactor: 'supermirror calibration',
+}
```

Comment:
Are these correction identifiers specified by Orso or are they just something we have to decide? Why is `total counts` a correction? :thinking:

Reply:
As far as I know, there is no specification. According to this https://www.reflectometry.org/file_format/specification, it is free form text.

> Why is total counts a correction? 🤔

Don't know. @arm61 specified it as such in the original package.

Reply:
My logic with `'total counts'` was that the data has been normalised by the total number of counts (if this isn't done, you don't have reflectivity, just reflected intensity).

Reply:
Sure. The question is more about what should count as a 'correction'.

Reply:
Okay, the standard says "list of reduction steps that have been performed" so that kind of makes sense, even if I find it a bit odd to call that a correction.

By the way now that you mentioned that the end result is reflectivity: I probably named the end-result wrongly when I ported this to sciline, the end result should probably be called RofQ instead of IofQ.